deploy/crds/push_v1alpha1_androidvariant_crd.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: androidvariants.push.aerogear.org
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.variantId
+    name: VariantId
+    type: string
+  - JSONPath: .status.ready
+    name: Ready
+    type: string
+  group: push.aerogear.org
+  names:
+    kind: AndroidVariant
+    listKind: AndroidVariantList
+    plural: androidvariants
+    singular: androidvariant
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            description:
+              description: Some description of the app.
+              type: string
+            senderId:
+              description: The "Google Project Number" (or Sender ID) from the API
```

Comment:
I notice that the auto-generated fields above follow the go doc standard for their description fields. E.g. "`thing` \<verb> [...]". Should these follow the same pattern?

Reply:
I took those descriptions from UPS code: https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/AndroidVariant.java#L42

Can be improved of course.

Reply:
I'd suggest maybe something like:

> senderId is the "Google Project Number" [...]"

wdyt?